### PR TITLE
DDF-3973 upgrade to camel 2.22.0 and artemis 2.6.2

### PR DESCRIPTION
**Pull request by ryeats**
#### Who is reviewing it? 
@harrison-tarr @roelens8 @kalcramer @derekwilhelm 
#### Select relevant component teams: 
https://github.com/orgs/codice/teams/core-apis
#### Ask 2 committers to review/merge the PR and tag them here.
@brendan-hofmann
@clockard
#### How should this be tested?
camel upgraded its file component so we may want to do some additional testing of the directory monitor.
#### What are the relevant tickets?
[DDF-3973](https://codice.atlassian.net/browse/DDF-3973)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: features/pom.xml [hunk 1]
```diff
@@ -51,28 +51,36 @@
         - Replaced the value of "geronimo-jms2-spec-version" with "${geronimo-jms_2_spec.version}" to keep it in line with the rest of the project
         - Added netty41 version and removed netty40 version
         - Updated qpid-proton-j-bundle-version and qpid-jms-client-version to 0.19.0 and 0.23.0 respectively-->
+        <aws-java-sdk-bundle-version>1.11.344_1</aws-java-sdk-bundle-version>
```
Comment (clockard) [inline features/pom.xml]: We should update the comment above about the camel version

File: features/camel/src/main/feature/feature.xml [hunk 2]
```diff
@@ -46,14 +46,17 @@
     DO NOT EDIT
     -->
     <!--We don't want this pulling in additional XML libraries on top of the ones already installed by CXF-->
-    <!--    <feature name="xml-specs-api" version="${servicemix-specs-version}" start-level="10">
+    <!--
+    <feature name="xml-specs-api" version="${servicemix-specs-version}" start-level="10">
```
Comment (clockard) [inline features/camel/src/main/feature/feature.xml]: Above comment should be updated with the new version

File: features/camel/src/main/feature/feature.xml [hunk 3]
```diff
@@ -165,9 +144,11 @@
         <feature version="${cxf-version-range}">cxf-core</feature>
         <feature version="${cxf-version-range}">cxf-jaxrs</feature>
         <feature version="${cxf-version-range}">cxf-jaxws</feature>
+        <!--<feature version="${cxf-version-range}">cxf-http-provider</feature>-->
```
Comment (clockard) [inline features/camel/src/main/feature/feature.xml]: Does this need to be commented out? A comment in the file as to why might be good.

Comment (ryeats) [reply features/camel/src/main/feature/feature.xml]: its what we did before ill add it to the comment block above.

**Comment by ryeats** [comment]
build now

**Comment by roelens8** [comment]
Do all of the jolokia calls still remain the same as before?  (ex. using jolokia to see the value of ReplicaSync or Topology) Wondering whether or not there were changes to the Artemis Mbeans in this new version of Artemis as sometimes they change how the URLs are structured. If so, we'd need to make sure the documentation is still up to date.

**Comment by ryeats** [comment]
build now

**Comment by ryeats** [comment]
build now

**Comment by KalCramer** [comment]
build now

**Comment by ryeats** [comment]
build now

**Comment by ryeats** [comment]
build now

**Comment by clockard** [comment]
Owasp failure for spring framework

**Comment by ryeats** [comment]
build now

**Pull request merged**